AppSettingsManagerAPI/AppSettingsManagerApi.Infrastructure/MySql/MySqlBaseUserRepository.cs

```diff
@@ -37,4 +37,20 @@ public async Task<Model.BaseUser> CreateUser(string userId, string password)
             _settingsContext.BaseUsers.Single(u => u.UserId == userId)
         );
     }
+    public async Task<Model.BaseUser> DeleteUser(string userId)
+    {
+        var user = await _settingsContext.BaseUsers.SingleAsync(u => u.UserId == userId);
+
+    _settingsContext.BaseUsers.Remove(user);
+    await _settingsContext.SaveChangesAsync();
+    return _baseUserConverter.Convert(user);
```

Comment:
```suggestion
        var user = await _settingsContext.BaseUsers.SingleAsync(u => u.UserId == userId);

        _settingsContext.BaseUsers.Remove(user);
        await _settingsContext.SaveChangesAsync();
        return _baseUserConverter.Convert(user);
```

@n898c Lol sorry, one last thing